tests/src/podman-extension-smoke.spec.ts

```diff
@@ -0,0 +1,127 @@
+import type { DashboardPage } from './model/pages/dashboard-page';
+import { expect as playExpect } from '@playwright/test';
+import { NavigationBar } from './model/workbench/navigation';
+import type { SettingsBar } from './model/pages/settings-bar';
+import { SettingsExtensionsPage } from './model/pages/settings-extensions-page';
+import { ExtensionPage } from './model/pages/extension-page';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let dashboardPage: DashboardPage;
+let settingsBar: SettingsBar;
+let navigationBar: NavigationBar;
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoName('settings-extensions-e2e');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navigationBar = new NavigationBar(page);
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+});
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+describe('Verification of Podman extension', async () => {
+  test('Podman is enabled and present', async () => {
+    await verifyPodmanExtensionStatus(true);
+  });
+  test('Disable Podman extension from Settings -> Extensions -> Podman page and verify status', async () => {
+    await openSettingsExtensionsPage();
+    const podmanExtensionPage = await openSettingsExtensionsPodmanPage();
+    await podmanExtensionPage.disableButton.click();
+    await verifyPodmanExtensionStatus(false);
+  });
+  test('Re-Enable Podman extension from Settings -> Extensions page and verify podman status', async () => {
+    const settingsExtensionsPage = await openSettingsExtensionsPage();
+    const podmanExtensionRowLocator = settingsExtensionsPage.getExtensionRowFromTable('Podman (default extension)');
+    await settingsExtensionsPage.getExtensionStartButton(podmanExtensionRowLocator).click();
+    await verifyPodmanExtensionStatus(true);
+  });
+});
+
+async function verifyPodmanExtensionStatus(enabled: boolean) {
+  dashboardPage = await navigationBar.openDashboard();
+  const podmanProviderLocator = dashboardPage.getPodmanStatusLocator();
+  enabled
+    ? await playExpect(podmanProviderLocator).toBeVisible()
+    : await playExpect(podmanProviderLocator).not.toBeVisible();
+  // always present and visible
+  const settingsExtensionsPage = await openSettingsExtensionsPage();
+  const settingsExtensionsPodmanLocator = settingsExtensionsPage.getFeaturedExtension('Podman');
+  await playExpect(settingsExtensionsPodmanLocator).toBeVisible();
+  const podmanExtensionRowLocator = settingsExtensionsPage.getExtensionRowFromTable('Podman (default extension)');
+  await playExpect(podmanExtensionRowLocator).toBeVisible();
+  const connectionStatusLabel = podmanExtensionRowLocator.getByLabel('connection-status-label');
+  await playExpect(connectionStatusLabel).toBeVisible();
+  const connectionStatusLocatorText = await connectionStatusLabel.innerText();
```

Comment:
Please add a timeout in the options object. default is quite long. 2/3sec perhaps.

Reply:
fixed